react/features/prejoin/components/preview/DeviceStatus.js

```diff
@@ -29,6 +31,43 @@ export type Props = {
     t: Function
 };
 
+const useStyles = makeStyles(theme => {
+    return {
+        deviceStatus: {
+            alignItems: 'center',
+            color: '#fff',
+            display: 'flex',
+            fontSize: '14px',
+            lineHeight: '20px',
+            padding: '6px',
+            textAlign: 'center',
+
+            '& span': {
+                marginLeft: theme.spacing(3)
+            },
+
+            '&.device-status-error': {
+                alignItems: 'flex-start',
+                backgroundColor: theme.palette.warning01,
+                borderRadius: '6px',
+                color: theme.palette.uiBackground,
+                padding: '12px 16px',
+                textAlign: 'left'
+            },
+            '& .device-icon': {
+                backgroundPosition: 'center',
+                backgroundRepeat: 'no-repeat',
+                display: 'inline-block',
+                height: '16px',
+                width: '16px'
+            },
+            '& .device-icon--ok svg path': {
+                fill: theme.palette.switch01Enabled
```

Comment:
I'm not sure whether or not this is okay. Maybe it would be better to use success color instead.

Reply:
Yes, please use a success color that's close to the original. Sorry I didn't check before

Reply:
`success04` from `colors` which is defined in `Tokens.js` is not accessible through theme. I used a HEX value as color. Maybe it would be a good idea to set `success` color in mui theme; So success, warning, error colors would be available on palette.